Lab 3

d3nik opened the pull request.

d3nik committed:
patch 3.1 "modern-normalize"

– "modern-normalize" connected;
– fixed list tag </ul>.

d3nik committed:
Patch 3.2

– layout of a page have been done;
– fixed a few errors in a code.

vozniuk-lntu commented on css/styles.css:
```diff
@@ -17,12 +18,65 @@ body ul {
     list-style: none;
```
онули тут маржини

vozniuk-lntu reviewed (changes requested):
1. нормалайз просто закинь в папку css
2. .header має бути 80px, вбільш висоту логотипа
3. .header .container прибрати спейс бітвін, підправити нав https://prnt.sc/92keOT3858vf, прибрати падіни з контактних ссилок, застосувати автоматичний лівий маржин
4. intro h1 відцентруф маржинами, intro на інтро не треба
5. не перебивай дефолтний контейнер https://prnt.sc/RiUDkSe3DTfj
6. очистит крайню геометрію (маржин) https://prnt.sc/8gHokAuTFe5Y
7. .occupation .container отут флекс не треба, заголовок вирівнюєть по центру текст алайном
8. прибрати max-width: 1170px; 
9. в атрибутах картинки гирина має бути не у пікселях
10. .footer-contacts має бути маржин топ для відступа до логотипу
11. .footer-contacts там не треба флекс, city-address додай нижній маржин, перербити контактні ссилки у футері, там накладаються маржини
12. projects має мати падінги вверж і вниз
13. .filters button прибрати маржини, відтуп між кнопками зроби через ГАП
14. почитай про gap для флекса
15. portfolio там використати гап, прибрати мардин в лішок

d3nik committed:
Patch 3.3 (hotfix)

– modern-normalize relocated to folder "css"
– fixed layout of header and footer

d3nik committed:
Patch 3.3.1 (hotfix)

– width of images in team card-list fixed;
– some max-width deleted;
– fixed some line-height;
– deleted some piece of useless code;
– added flex gap instead of margins;

vozniuk-lntu reviewed (approved):
1. отут .footer-contacts ul li використати :not(:firs-child)
2. cards-list нема GAP

d3nik committed:
Patch 3.3.2 (hotfix)

– cards-list now have gap;
– .footer-contacts ul li using :not(:last-child), .city-address now displaying as block and have own margin.
